onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -184,6 +184,16 @@ def _cat_input_wrangler(
     return args, kwargs
 
 
+def _cross_entropy_input_wrangler(
+    args: list[Any], kwargs: dict[str, Any]
+) -> tuple[list[Any], dict[str, Any]]:
+    # Remove the self argument
```

Comment:
What does this comment mean? I see the code is trying to update the data type of weight, not removing any other ones.

Reply:
fixed the comment